### ISSUE TITLE
[Feature Request] Toggle GM Screen via Macro

### ISSUE THREAD

**Issue by PlasticMohawk** [angometry/trinium-chat-buttons#10]
As a Foundry GM, I want to be able to open and close the GM Screen without having to have the chat window actively selected - for example, via macro on my macro bar, and/or hotkey.

Thanks in advance for considering!

**Comment by angometry** [comment]
Absolutely! haven't had time to work on it for a couple weeks but the hotkey is the very next feature on a long list.

Api for interacting through code and macros will eventually come, but not too soon

**Comment by angometry** [comment]
Added in 1.6.0, with a configurable button (default G) to toggle GM screen
